### PR TITLE
Remove old points label instead of updating labels

### PR DESCRIPTION
**Pull request by BunnyDunker**
Updating the labels would cause all labels to be deleted and replaced with the new one

### CODE REVIEW

**Review by adam-hebert656 — APPROVED** [review]

File: lib/slax/poker.ex [hunk 1]
```diff
@@ -63,15 +63,33 @@ defmodule Slax.Poker do
...
+         {200, _issue, _http_response} <-
+           Tentacat.Issues.Labels.add(client, org, repo, issue, ["Points: #{score}"]) do
+      :ok
+    else
+      {_response_code, %{"message" => error_message}, _http_response} ->
+        {:error, error_message}
+    end
+  end
+
+  defp maybe_remove_label(client, org, repo, issue, labels) do
```
Comment (grossvogel) [inline lib/slax/poker.ex:79]: I think there are cleaner / more readable ways to express this same logic... one idea would be to have a separate function like `find_points_label(labels)` and call it up in your main `decide` function. Then you can have a `maybe_remove_label(client, org, repo, issue, nil), do: :ok` clause that doesn't do anything and a `maybe_remove_label(client, org, repo, issue, label) do...` function that actually does the work. That one could just do a case statement on the tentacat response.

**Review by grossvogel — APPROVED** [review]
One idea for cleaning the code up a bit, but not a blocker

File: lib/slax/poker.ex [hunk 2]
```diff
@@ -63,15 +63,41 @@ defmodule Slax.Poker do
...
+    with %{"name" => label} <-
+           Enum.find(labels, fn
+             %{"name" => "Points: " <> _number} -> true
+             %{"name" => _} -> false
+           end) do
+      label
+    else
+      nil ->
+        nil
+    end
```
Comment (grossvogel) [inline lib/slax/poker.ex:90]: One very general rule of thumb: `with` is best reserved for things that can fail. Other things like `if`, `case`, and `cond` are better for control flow, in that order of preference usually (using the simplest one that works). In this case, I think it's more readable like this

```suggestion
    points_label = Enum.find(labels, fn
      %{"name" => "Points: " <> _number} -> true
      _ -> false
    end)
    
    if points_label, do: points_label["name"], else: nil
```

**Review by grossvogel — APPROVED** [review]
Another totally style-focused nit 😄  (untested suggestion alert!)

**Comment by BunnyDunker** [comment]
I'm not sure how this happened, I'd have to go digging in to commit history, but this functionality is already on master so I'm going to close this PR

**Pull request closed without merging**